gum/backend-darwin/gumkernel-darwin.c

```diff
@@ -704,6 +761,12 @@ gum_kernel_bruteforce_base (GumAddress unslid_base)
    */
 
   gint slide_byte;
+  bool is_debug;
```

Comment:
For consistency we should avoid C99 types:

```suggestion
  gboolean is_debug;
```